### PR TITLE
Module 1: Portfolio

### PR DESCRIPTION
**Pull request by oluwatobi-fasasi**
Add works section
Add about section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,11 +16,113 @@
...
-        <section>
+        <section class="intro">
             <h1>Fasasi Abdulwasiu</h1>
             <p class="para">Hello! I am a software developer! I can help you build a product, feature or website. Take a look at my work. <br><br> If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
             <a href="https://microverse.org" class="button">Start collaboration</a>
         </section>
+        <section class="project">
+            <!-- <div class="rectangle"></div> -->
+            <h2>Projects</h2>
+            <section class="works">
```
Comment (Ibizugbe) [inline index.html:27]: Nice work so far👍 but currently, there is a part of the design missing😉 kindly take note of the highlighted part in the image😉 don't forget, It is a requirement that we stick to the design in this milestone🤗🤗

absent_section1

kindly add that part of the design to your project🤗🤗

File: index.html [hunk 2]
```diff
@@ -16,11 +16,113 @@
...
+                <p>Skills</p>
+                <ul>
+                    <li>Codekit</li>
+                    <li>Codepen</li>
+                    <li>GitHub</li>
+                    <li>GitLab</li>
+                    <li>Terminal</li>
+                </ul>
+            </div>
+        </section>
```
Comment (Ibizugbe) [inline index.html:125]: Nice work building the `b=about`  section😉 but there our still some things making our project different from the given design😉😉

Your about
your__Special image



Design
design__specual images

kindly see the images absent, and notice the difference in spacings and fix the issue🤗

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @Fasasi 👋👋, 

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f69656d6d596a3332456945655a59766a70712f67697068792e676966


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅

- [ ] Linters are passing✅
- [ ] Github flow has been used✅


## Required Changes ♻️

- [ ] As specified in the previous milestone of the project, the mobile screen size should be from `375px` this is affecting the responsiveness of your project😉 I suggest you kindly fix it🤗

Your code
375px width

appearance
inresponsive

kindly adjust the width😉

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [OPTIONAL] Kindly ensure the contents of your page are wrapped in a `main` tag as this makes your page more semantic😉 The previous reviewers may have forgotten to point it, but it is a requirement🤗🤗

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by oluwatobi-fasasi** [comment]
Hello,

The images on my about page are not showing. I downloaded the PNG file on
my local system but it is not showing up on my work.

That is why I ignored it earlier.

Thank you
ᐧ
ᐧ

**Comment by oluwatobi-fasasi** [comment]
Thanks, I just found my way around it.
ᐧ

File: index.html [hunk 3]
```diff
@@ -8,19 +8,124 @@
...
 <body>
-    <div class="main-wrapper">
+    <main>
         <header>
             <div>
                 <img src="images/Shape.png" alt="shape" class="shape">
                 <a href="#index.html"><img src="images/Shape (1).png" alt="site-logo" class="logo"></a>
             </div>
             <img src="images/ic_menu.svg" alt="navbar-menu" class="menu">
         </header>
```
Comment (bobb-Rob) [inline index.html:18]: - [ ] [OPTIONAL] The hamburger/menu icon is touching the right edge of the screen. Please fix this.
image

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @oluwatobi-fasasi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 4]
```diff
@@ -66,10 +66,198 @@ section a {
...
+.intro a:active {
   background-color: #0048fd;
 }
+
+/* .rectangle {
+  background-color: rgb(119, 84, 84);
+  width: 320px;
+  height: 160px;
+  margin-top: 30px;
+} */
```
[the remark this answers is not in the document]
Comment (oluwatobi-fasasi) [reply styles.css]: Thanks @Ibizugbe, all suggestion changes are fully implemented

File: index.html (hunk 3 above)
Comment (oluwatobi-fasasi) [reply index.html:18]: Thanks @bobb-Rob, all suggestion changes are fully implemented

**Pull request merged**